examples/flower-client/client/flwr_task.py

```diff
@@ -92,4 +92,4 @@ def get_weights(net):
 def set_weights(net, parameters):
     params_dict = zip(net.state_dict().keys(), parameters)
     state_dict = OrderedDict({k: torch.tensor(v) for k, v in params_dict})
-    net.load_state_dict(state_dict, strict=True)
\ No newline at end of file
+    net.load_state_dict(state_dict, strict=True)
```

Comment:
Is this from the Flower example or did we add it? We should have unmodified flwr_task.py

Reply:
good